### DISCUSSION TITLE
Can't buy or sell claim blocks

### DISCUSSION THREAD

**Discussion by Wolfy8198** [GriefPrevention/GriefPrevention#1388] [Q&A] [answered]
I can't buy or sell claim blocks on my server. However, when I run another localhost server with only EssentialsX, GriefPrevention, and Vault, it allows me to buy and sell claim blocks. I have prices set for buying and selling claim blocks, but it keeps telling me that buying and selling claim blocks is disabled.
I have my config pasted here: https://pastebin.com/Dmqh14zy
and I have a screenshot of the plugins I have installed attacked to this (document?)
![Screenshot 2021-06-01 150807](https://user-images.githubusercontent.com/64490202/120377539-407d7900-c2eb-11eb-8d1a-f81d20c2e4ac.png)
Is a plugin I have installed blocking me from buying claim blocks? If not, why can't I get the bonus claim blocks?

**Comment by bigpresh** [comment]
Anything in your server log when GP loads regarding enabling economy integration with Vault?

I'd try temporarily starting up without all the other plugins (just GP and Vault) and see if the problem still happens; if it does then it's a config issue or something between GP and Vault, but if the issue goes away, you'll know it's a plugin.  Looking at that list, TradeSystem and Shopkeepers sound like ones I'd suspect first.

(I know you've said you've tried just GP and Vault on another server elsewhere, but I'd be testing the same server, so you know it's the same plugin versions, same configs etc)

**Reply by Wolfy8198** [reply]
I copied everything regarding those plugins to the localhost server, and I didn't change anything. Versions, config files, everything was copied. So you think it's a plugin that's blocking it?

**Comment by Jikoo** [comment, marked as answer]
Could you post server startup logs from startup to an attempt at using the commands? Remove your IP during login if the server's not local of course. GP should be logging Vault's state when the bridge loads.

**Reply by Wolfy8198** [reply]
I think I got it fixed somehow. I disabled Shopkeepers, tried the command again, and it worked, ro I enabled shopkeepers again and it still worked. I think I just needed to restart the server for the effect of the config change to connect with Vault
